components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -152,10 +155,12 @@ class AddonsManagerAdapter(
     }
 
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
-    internal fun bindSection(holder: SectionViewHolder, section: Section) {
+    internal fun bindSection(holder: SectionViewHolder, section: Section, position: Int) {
         holder.titleView.setText(section.title)
+        holder.divider.isVisible = style?.visibleDividers != null && position != 0
```

Comment:
Nit: Either if `visibleDividers` is `true` or `false`  `style?.visibleDividers != null` will be true.Not sure if this is the intention.

Reply:
Sorry, it was `dividerColor` in the first variant of this PR, and I forgot to change the check.